p2p/src/net/mock/peer.rs

```diff
@@ -13,18 +13,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::sync::Arc;
+
+use futures::FutureExt;
+use tokio::sync::mpsc;
+
+use common::chain::ChainConfig;
+use logging::log;
+
```

Comment:
why are there empty lines between these?

Reply:
Personally I prefer to separate imports into a several groups: std, third-party crates, workspace crates and imports from the current crate. We don't have such convention, so I usually try not to introduce unnecessary changes, but I got carried away with this pull request. I can revert these changes if you don't like them.